device/arista/x86_64-arista_7060x6_64pe_b/Arista-7060X6-64PE-B-O128/pg_profile_lookup.ini

```diff
@@ -0,0 +1,4 @@
+# PG lossless profiles.
+# speed cable size    xon  xoff threshold xon_offset
+400000  5m    18796   0    445770  0         3556
+400000  40m   18796   0    445770  0         3556
```

Comment:
Need another cable length between LT2 and UT2

Reply:
There are 3 different cable length
- 5m
- 30m
- 500m
@kperumalbfn Please double confirm. Thanks